benchmarks/benchmarks/hungarian.py

```diff
@@ -0,0 +1,27 @@
+from numpy.random import rand
+
+try:
+    from scipy.optimize._hungarian import linear_sum_assignment
+except ImportError:
+    pass
+
+from .common import Benchmark
+
+
+class Hungarian(Benchmark):
+    """
+    Test the speed of the Munkres algorithm for solving the
+    assignment problem
+    """
+    params = [
+        [100, 300, 500, 700]
+    ]
+
+    param_names = ['matrix_size']
+
+    def setup(self, matrix_size):
+        self.cost_matrix = rand(matrix_size, matrix_size)
```

Comment:
you need to use a seed here, or the values will change run to run which will affect the timings.

Reply:
also would be good to add the benchmark results (comparing to master) to the PR description

Reply:
I have made the requested changes. Although I may need to push a new commit as the tests are currently failing due to an unresponsive server somewhere.

Reply:
don't worry about those test failures; they're known issues. Due to the way CircleCI is set up you'd need to rebase on latest master anyway to get those builds green.